### ISSUE TITLE
XML output shows incorrect freespace

### ISSUE THREAD

**Issue by Thunderbottom** [nextcloud/serverinfo#223]
### Steps to reproduce
1. Visit `/ocs/v2.php/apps/serverinfo/api/v1/info` after login

### Expected behaviour
Correctly reported `freespace` value

### Actual behaviour
`freespace` value shows up as `-2`. Example output:

```xml
<enable_avatars>yes</enable_avatars>
<enable_previews>yes</enable_previews>
<memcache.local>\OC\Memcache\APCu</memcache.local>
<memcache.distributed>\OC\Memcache\Redis</memcache.distributed>
<filelocking.enabled>yes</filelocking.enabled>
<memcache.locking>\OC\Memcache\Redis</memcache.locking>
<debug>no</debug>
<freespace>-2</freespace>
<cpuload>
</cpuload>
<mem_total>7936524</mem_total>
<mem_free>6946508</mem_free>
<swap_total>0</swap_total>
<swap_free>0</swap_free>
```

The account being used is an admin account with no quota limits. Also, just a heads up that the nextcloud instance is being served from inside a docker container.

### Server configuration

**Operating system:** Nextcloud Official Docker container

**Web server:** Apache

**Database:** Postgres

**PHP version:**

**Nextcloud version:** 17.0.1

**Where did you install Nextcloud from:** `nextcloud:17.0.1-apache` from dockerhub

**Comment by joshtrichards** [comment]
Is this still reproducible today?

**Comment by joshtrichards** [comment]
This was fixed in #294.
